pkg/hub/client/public/facade/doc.go

```diff
@@ -0,0 +1,3 @@
+// Package facade groups dedicated function per kind.
```

Comment:
I need to write it: I don't like the package with such name 😄  especially if currently we have just `additional_refs.go` inside the `facade/types` package. For me it's unnecessary directory structure and we can make them flat.

Can we delete this package, and move `facade/types/additional_refs.go` to `type_additional_refs.go` on the root of `pkg/hub/client/public`?

Reply:
IMO It will be problematic on long run as it mixes two different things. For now, I made it flat, but I would like to follow up on this 🙂

Reply:
I agree - but then I believe we would extract more files to subpackages (e.g. `SortImplementationRevisions` etc.). We'll revisit that next time 👍